app/src/server/tasks/generateTestSetEntry.task.ts

```diff
@@ -125,21 +125,22 @@ export const generateTestSetEntry = defineTask<GenerateTestSetEntryJob>({
       }
 
       let completion: ChatCompletion;
-      const input: ChatCompletionCreateParams = {
+      const input: ChatCompletionCreateParamsNonStreaming = {
         model: fineTune
           ? `openpipe:${fineTune.slug}`
           : COMPARISON_MODEL_NAMES[modelId as ComparisonModel],
         messages: datasetEntry.messages,
         tool_choice: datasetEntry.tool_choice ?? undefined,
         tools: datasetEntry.tools ?? undefined,
         response_format: datasetEntry.response_format ?? undefined,
+        stream: false,
       };
 
       try {
         if (isComparisonModel(modelId)) {
           completion = await getOpenaiCompletion(rawDatasetEntry.dataset.projectId, input);
         } else if (fineTune) {
-          completion = await getCompletion2(fineTune, input);
+          completion = (await getCompletion2(fineTune, input)) as ChatCompletion;
```

Comment:
If we make the change to the typings of `getCompletion2` we can drop this. If we don't though should make this an actual check on the return type, not a type assertion. Too easy to make make mistakes with type assertions!